src/Admin/BaseMediaAdmin.php

```diff
@@ -37,6 +37,9 @@ abstract class BaseMediaAdmin extends AbstractAdmin
 
     protected ?ContextManagerInterface $contextManager = null;
 
+    /**
+     * @var string
```

Comment:
Introduced by https://github.com/sonata-project/SonataMediaBundle/pull/2035/files#diff-f7d08d0a17e3f0b38e23f90b6a73f036621412f2e2c74be531256496e1715ac8R45

best would be to remove this phpdoc to rely on the default one.